src/components/main/navbar/index.tsx

```diff
@@ -33,7 +37,7 @@ function NavBar(props: Props) {
       <div className={styles.sortBySelect}>
         <p className={styles.secondaryText}>Sort by</p>
         <div className={styles.selectContainer}>
-          <select className={styles.selectInput} value={sortBySelected} onChange={(e: React.ChangeEvent<HTMLSelectElement>) => setSortBySelected(e.target.value)}>
+          <select className={styles.selectInput} value={sortBySelected} onChange={(e: React.ChangeEvent<HTMLSelectElement>) => dispatch(setSortBySelectedAction(e.target.value))}>
```

Comment:
type is redundant here
```suggestion
          <select className={styles.selectInput} value={sortBySelected} onChange={e => dispatch(setSortBySelectedAction(e.target.value))}>
```
```suggestion
          <select className={styles.selectInput} value={sortBySelected} onChange={(e: React.ChangeEvent<HTMLSelectElement>) => dispatch(setSortBySelectedAction(e.target.value))}>
```

Reply:
Got it